kuka_eki_hw_interface/CMakeLists.txt

```diff
@@ -65,3 +65,5 @@ install(
 install(
   TARGETS kuka_eki_hw_interface_node
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+
+install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

Comment:
This is a driver. I don't expect anyone to want to link against the hw interface, so I'm not sure the headers need to be installed in this case. Even with combined hw interfaces (for which this driver would need to be updated), you don't need the headers.

We should probably remove the `INCLUDE_DIRS` and `LIBRARIES` from `catkin_package(..)` instead.